node-src/lib/getDependentStoryFiles.test.ts

```diff
@@ -221,7 +221,7 @@ describe('getDependentStoryFiles', () => {
     const changedFiles = ['src/foo.css'];
     const modules = [
       {
-        id: null,
+        id: '',
```

Comment:
More of the same here, this is meant to represent the real world input format, so I think it should get a pass on our eslint rules.

Reply:
Are you thinking the same for tests starting at line 617?

Reply:
To play it safe, I brought those back too!